rest-api/dao/physical_measurements_dao.py

```diff
@@ -241,7 +247,11 @@ def insert_with_session(self, session, obj):
           return measurements
     PhysicalMeasurementsDao.set_measurement_ids(obj)
 
-    super(PhysicalMeasurementsDao, self).insert_with_session(session, obj)
+    inserted_obj = super(PhysicalMeasurementsDao, self).insert_with_session(session, obj)
+    if not is_amendment:  # Amendments aren't expected to have site ID extensions.
+      ParticipantDao().add_missing_hpo_from_site(
```

Comment:
Are we positive that all new physical measurements will have these extensions? (If you haven't already, please check with Jason Tan.)